### ISSUE TITLE
Issues with generating Asset Packs and Building

### ISSUE THREAD

**Issue by s6-willard** [jelte/be.khepri.play.assetdelivery.addressables#10]
1. Bundles for the addressables are generated in `Library\com.unity.addressables\StreamingAssetsCopy\aa\Android\Android`, but the plugin only looks for bundles in `Library\com.unity.addressables\StreamingAssetsCopy\aa\Android`, (note the missing Android directory at the end), which is why there are no packs being generated.

2. This is mentioned somewhere else, but the Build App Bundle isn't working. Somehow the Google Play PAD plugin still using the bundle name with the ".bundle" extension at the end, which is not a valid asset pack name.

**Comment by jelte** [comment]
1. this is the code used, not sure why it's showing a double Android for use as it uses the `Addressables.BuildPath` property.  
```
internal class AddressablesImpl
{
        ...
        public string BuildPath
        {
            get { return "Library/com.unity.addressables/StreamingAssetsCopy/" + StreamingAssetsSubFolder + "/" + PlatformMappingService.GetPlatform(); }
        }
        ...
}
```
What version of addressables are you using?

2. sorry, I've been fairly busy with work to address the PR's and Issues I've seen myself. I'll try to get on them.

**Comment by s6-willard** [comment]
Thanks for replying!

1. It looks like my build path ultimately is "[UnityEngine.AddressableAssets.Addressables.BuildPath]/[BuildTarget]". Not sure if this was changed before or if this is the default. In the meantime I can do a workaround for this.

2. Yeah I noticed as well that the bundles have to be copied without the extension. I see that that is what #7 is doing, so I can just wait for that.

Thank you!
